src/Firebase.Notifications.Android/Common.uno

```diff
@@ -16,11 +16,13 @@ namespace Firebase.Notifications
     [extern(iOS) Require("Source.Include", "Firebase/Firebase.h")]
     public static class NotificationService
     {
+        /*  
         extern(!Android)
         static NotificationService()
         {
             OnRegistrationFailed(null, "Firebase Notifications are not yet available on this platform");
         }
+        */
```

Comment:
Please delete the code instead of commenting it out. We have the git-history for people who want old versions.